SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -20,7 +20,7 @@
 
 DEFINE_LOG_CATEGORY(LogSpatialClassInfoManager);
 
-void USpatialClassInfoManager::Init(USpatialNetDriver* InNetDriver)
+bool USpatialClassInfoManager::Init(USpatialNetDriver* InNetDriver)
```

Comment:
I'm disliking that this isn't consistent with the rest of the `Init` signatures. Hmmm

Reply:
Can you rename this function to `TryLoadSchemaDatabase` instead of `Init`?